### ISSUE TITLE
Cython compilation failure error

### ISSUE THREAD

**Issue by frankgould** [matham/ffpyplayer#78]
I am trying to install ffpyplayer in Ubuntu Mate running under Virtualbox on Windows7 after installing buildozer in a virtual environment ([per these instructions](http://inclem.net/2019/05/19/kivy/running_buildozer_in_a_virtual_machine/)). The log at the link below shows a problem with setuptools but we confirmed it was installed correctly, we believe (@kuzeyron is the other half of 'we'). I also installed cython==0.29.9 per online docs and got same results. Not sure what I need to do to get around this cython compilation error.

https://gist.github.com/frankgould/bffcad93c6f2b29255aec8601ba2ecb0

**Comment by matham** [comment]
Please see https://stackoverflow.com/questions/34819221/why-is-python-setup-py-saying-invalid-command-bdist-wheel-on-travis-ci. This is not a ffpyplayer issue, but rather you either have a old pip, missing wheel, or something else in your environment is broken.

Please try the solutions suggested in that link, but this is ultimately not related to ffpyplayer afaict.

**Comment by frankgould** [comment]
I tried all the suggestions on that stackoverflow page and ffpyplayer could never install using the wheel. So, I searched for `ffpyplayer/pic.pyx:57:0: 'includes/inline_funcs.pxi' not found` and found the issue at the link below. I followed the procedures posted by this commenter and was able to install `ffpyplayer` successfully, as documented in the console log below. 
https://github.com/matham/ffpyplayer/issues/55#issuecomment-483724671

```
android@android-vmw7:~$ sudo pip3 install https://github.com/matham/ffpyplayer/archive/master.zip
Collecting https://github.com/matham/ffpyplayer/archive/master.zip
  Downloading https://github.com/matham/ffpyplayer/archive/master.zip
     \ 5.5MB 23.4MB/s
Installing collected packages: ffpyplayer
  Running setup.py install for ffpyplayer ... done
Successfully installed ffpyplayer-4.3.0.dev0
```
To do this, I first installed `buildozer` without a virtual environment, installed `kivy` (also no venv), then after trial and error compiles, I installed the following modules:  `libsdl2-dev, libavformat-dev,`and`libavdevice-dev` to get a successful install. I've tested the import of `.player`, `.write`, and `.pic`, but assume they will work. I believe `kivy` installed with a wheel but could never get `ffpyplayer` to build that way.

I'm documenting this in case others may find it useful.
